Bottom: 9fbb8863c1a7bb0c2535876922ebf95ed3509ed9
Top:    aebe22aadf3c48620d9af13ce8b12d27c2a87440
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:31 +0000

Remove old debug code for splicing a cylinder onto the board outline object3d


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 8a14671..f81d5c2 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -633,98 +633,6 @@ object3d_from_contours (POLYAREA *contours,
             }
         }
 
-      if (0)
-        {
-          /* Cylinder centers on 45x45mm, stitch vertex is at 40x45mm. Radius is thus 5mm */
-
-          edge_ref cylinder_edges[3];
-          vertex3d *cylinder_vertices[2];
-          face3d *cylinder_faces[2];
-
-          /* Edge on top of board */
-          cylinder_edges[0] = make_edge ();
-          UNDIR_DATA (cylinder_edges[0]) = make_edge_info ();
-#ifdef REVERSED_PCB_CONTOURS
-          edge_info_set_round (UNDIR_DATA (cylinder_edges[0]),
-                               COORD_TO_STEP_X (PCB, MM_TO_COORD (45.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 0., /* Center of circle */
-                                0.,   0., 1., /* Normal */
-                                5.);          /* Radius */
-#else
-          edge_info_set_round (UNDIR_DATA (cylinder_edges[0]),
-                               COORD_TO_STEP_X (PCB, MM_TO_COORD (45.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 0., /* Center of circle */
-                                0.,   0., 1., /* Normal */
-                                5.);         /* Radius */
-#endif
-          object3d_add_edge (object, cylinder_edges[0]);
-
-          /* Edge on top of cylinder */
-          cylinder_edges[1] = make_edge ();
-          UNDIR_DATA (cylinder_edges[1]) = make_edge_info ();
-          edge_info_set_round (UNDIR_DATA (cylinder_edges[1]),
-                               COORD_TO_STEP_X (PCB, MM_TO_COORD (45.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 10., /* Center of circle */
-                                0.,   0., 1.,  /* Normal */
-                                5.);          /* Radius */
-          object3d_add_edge (object, cylinder_edges[1]);
-
-          /* Edge stitching cylinder */
-          cylinder_edges[2] = make_edge ();
-          UNDIR_DATA (cylinder_edges[2]) = make_edge_info ();
-          edge_info_set_stitch (UNDIR_DATA (cylinder_edges[2]));
-          object3d_add_edge (object, cylinder_edges[2]);
-
-          /* Vertex on board top surface */
-          cylinder_vertices[0] = make_vertex3d (COORD_TO_STEP_X (PCB, MM_TO_COORD (40.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 0.); /* Bottom */
-          object3d_add_vertex (object, cylinder_vertices[0]);
-
-          /* Vertex on cylinder top surface */
-          cylinder_vertices[1] = make_vertex3d (COORD_TO_STEP_X (PCB, MM_TO_COORD (40.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 10.); /* Top */
-          object3d_add_vertex (object, cylinder_vertices[1]);
-
-          /* Cylindrical face */
-          cylinder_faces[0] = make_face3d ();
-          face3d_set_cylindrical (cylinder_faces[0], COORD_TO_STEP_X (PCB, MM_TO_COORD (45.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 0., /* A point on the axis of the cylinder */
-                                            0., 0., 1.,             /* Direction of the cylindrical axis */
-                                            5.);                   /* Radius of cylinder */
-          face3d_set_normal (cylinder_faces[0], 1., 0., 0.);       /* A normal to the axis direction */
-                                       /* XXX: ^^^ Could line this up with the direction to the vertex in the corresponding circle edge */
-          object3d_add_face (object, cylinder_faces[0]);
-          face3d_add_contour (cylinder_faces[0], make_contour3d (cylinder_edges[0]));
-
-          /* Top face of cylinder */
-          cylinder_faces[1] = make_face3d (); /* top face of cylinder */
-          face3d_set_normal (cylinder_faces[1], 0., 0., 1.);
-          face3d_set_appearance (cylinder_faces[1], top_bot_appearance);
-          object3d_add_face (object, cylinder_faces[1]);
-          face3d_add_contour (cylinder_faces[1], make_contour3d (cylinder_edges[1]));
-
-          /* Splice onto board */
-          face3d_add_contour (faces[npoints + 1], make_contour3d (SYM(cylinder_edges[0])));
-
-          /* Assign the appropriate vertex geometric data to each edge end */
-          ODATA (cylinder_edges[0]) = cylinder_vertices[0];
-          DDATA (cylinder_edges[0]) = cylinder_vertices[0];
-          ODATA (cylinder_edges[1]) = cylinder_vertices[1];
-          DDATA (cylinder_edges[1]) = cylinder_vertices[1];
-          ODATA (cylinder_edges[2]) = cylinder_vertices[0];
-          DDATA (cylinder_edges[2]) = cylinder_vertices[1];
-          LDATA (cylinder_edges[0]) = cylinder_faces[0];
-          RDATA (cylinder_edges[0]) = faces[npoints + 1]; /* TOP OF BOARD FACE */
-          LDATA (cylinder_edges[1]) = cylinder_faces[1];
-          RDATA (cylinder_edges[1]) = cylinder_faces[0];
-          LDATA (cylinder_edges[2]) = cylinder_faces[0];
-          RDATA (cylinder_edges[2]) = cylinder_faces[0];
-
-          /* Splice things together.... */
-
-          /* Link edges orbiting the cylinder bottom vertex */
-          splice (cylinder_edges[0], cylinder_edges[2]);
-          splice (cylinder_edges[2], SYM(cylinder_edges[0]));
-
-          /* Link edges orbiting the cylinder top vertex */
-          splice (SYM(cylinder_edges[2]), cylinder_edges[1]);
-          splice (cylinder_edges[1], SYM(cylinder_edges[1]));
-        }
-
 #ifndef NDEBUG
       ct = contour;
       start_of_ct = 0;
